convert environment0000.txt to snellius/ossc

f-hafner opened the pull request:
This pull request adds requirements files for copying the venv for the summer school from the RA to the OSSC.

Instructions. On a snellius login node, run 
```
source ossc/modules.sh
```
this creates the following files:
- `requirements_ossc.txt`: file to share with SURF for setting up the venv
- `snellius_missing`: the packages that are in `requirements0000.txt` but not in the snellius environment

Once SURF has installed the venv, one should be able to activate the entire environment with
```bash
module load xyz # load all modules that have `module load` in the the file `ossc/modules.sh`
source ossc_env/bin/activate
```

f-hafner committed:
convert environment0000.txt to snellius/ossc

f-hafner commented:
for duckdb, do we need other dependencies, ie the executable mentioned [here](https://duckdb.org/docs/installation/?version=stable&environment=cli&platform=linux&download_method=package_manager)?

jgarciab commented:
This looks great @f-hafner! I wonder if it's worth separating the libraries required for data processing and modeling (which we need at OSSC) from jupyter and the libraries required from visualization (which we don't need at OSSC). Regarding duckdb, we could also not support it. I doubt people will end up using it.

f-hafner committed:
remove duckdb/jupyter; compare version numbers

f-hafner commented:
@jgarciab , I removed duck db and the jupyter dependencies. I think it could be handy to plot some learning curves or similar directly on the ossc, so I suggest keeping the plotting libraries.
I also added a csv file to compare the versions of installed packages in both environments

f-hafner committed:
add pyreadstat
